processing/src/main/java/io/druid/query/expression/ExprUtils.java

```diff
@@ -56,20 +58,21 @@ public static PeriodGranularity toPeriodGranularity(
     final DateTime origin;
     final DateTimeZone timeZone;
 
-    if (originArg == null) {
-      origin = null;
-    } else {
-      final Object value = originArg.eval(bindings).value();
-      origin = value != null ? new DateTime(value) : null;
-    }
-
     if (timeZoneArg == null) {
       timeZone = null;
     } else {
       final String value = timeZoneArg.eval(bindings).asString();
       timeZone = value != null ? DateTimeZone.forID(value) : null;
     }
 
+    Chronology chronology = timeZone == null ? ISOChronology.getInstanceUTC() : ISOChronology.getInstance(timeZone);
```

Comment:
Chronology is only needed in the `else` case, right? can it be limited in scope to just that code block?